src/utils/option-deprecation.mjs

```diff
@@ -0,0 +1,31 @@
+import { chalk, log, NETLIFYDEVWARN } from './command-helpers.mjs'
+
+// This file contains logic for flags in camelcase that will be deprecated in v13 and replaced in kebab-case.
+// Flags to be deprecated in v13
+const DEPRECATED_OPTIONS = {
+  httpProxy: { option: '--httpProxy', newOption: '--http-proxy', deprecatedVersion: 'v13' },
```

Comment:
I'm not sure we should commit to a specific version of the CLI where we'll deprecate these flags. I think we'd want to leave a certain period of time for people to migrate, and it's totally possible that we have to fit in a breaking change and release v13 next week.

Could we just say we'll deprecate this in an upcoming version of the CLI?

Reply:
Yeah 100%!